networkx/algorithms/centrality/harmonic.py

```diff
@@ -0,0 +1,91 @@
+"""
+Harmonic centrality measure.
+"""
+#    Copyright (C) 2004-2015 by
+#    Alessandro Luongo
+#    BSD license.
+from __future__ import division
+import functools
+import networkx as nx
+
+
+__author__ = "\n".join(['Alessandro Luongo (alessandro.luongo@studenti.unimi.it'])
+__all__ = ['harmonic_centrality']
+
+
+def harmonic_centrality(G, u=None, distance=None):
+    """Compute harmonic centrality for nodes.
+
+    Harmonic centrality [1] of a node `u` is the sum of the reciprocal
+    of the shortest path distances from `n-1` other nodes to `u`.
```

Comment:
Is it simpler (and still accurate) to say: "...of node `u` is the sum of the reciprocal of the nonzero shortest path distances from all other nodes."?